src/Npgsql/NpgsqlBinaryImporter.cs

```diff
@@ -358,6 +364,19 @@ async Task WriteRow(bool async, params object[] values)
                 await Write(value, async);
         }
 
+        /// <summary>
+        /// Checks if the column to be written belongs to the allowed columns for this importer
+        /// </summary>
+        /// <param name="targetColumnIndex">The target column index to be written. <see cref="_column"/></param>
+        /// <param name="currentNumColumns">The number of columns for this binary importer. <see cref="NumColumns"/></param>
+        void CheckColumnInTable(short targetColumnIndex, int currentNumColumns)
+        {
+            if (targetColumnIndex >= currentNumColumns)
```

Comment:
Since the method is a instance method, you can already access `NumColumns` and `_column` from there method, passing them via arguments is excessive. Probably the method name should be changed to `CheckColumnIndex`, so it will be more obvious.

Reply:
Cool. I changed it to use the members available. Also renamed as you suggested.